### ISSUE TITLE
Extremly big project, but I need some contributors for it to finish!

### ISSUE THREAD

**Issue by Rix565** [pbsds/hatena-server#34]
I tried a long time ago to 'convert' the server from Python 2.7 to Python 3.8 (due to Python 2.7 being shut down)

For security reasons & to make this project more modern & more compatible with nowadays modules,

I want to restart this project again...

But because I was alone to do this, I failed to do it.

It's for this reason I need other contributors to do this.

Because if we are a few to work on it, we can finish this project and finally make this project more compatible with modules than ever!

**Comment by Logrythmic24** [comment]
Hey Rix! Are there any updates? :)

**Comment by Rix565** [comment]
@Logrythmic24 yeah! I rewrote literally hatena-server in django with python3 :^)

**Comment by Logrythmic24** [comment]
@Rix565 Awesome! any chance I could see it sometime?

**Comment by Rix565** [comment]
@Logrythmic24 https://github.com/Aftendo/Afternote
